source/androidx.compose.foundation/foundation/Transforms/Metadata.Namespaces.xml

```diff
@@ -0,0 +1,78 @@
+﻿<metadata>
+    <remove-node
+        path="/api/package[@name='androidx.compose.foundation']" 
+        />
+    <remove-node
```

Comment:
Why switch to this style from the previous?
```
<remove-node path="path=/api/package" />
```

This seems like more work, and will require us to manually update this with each new namespace added to future Compose versions.

Reply:
Remember I asked have you ever tried that XPath which caused:

```
BINDINGSGENERATOR : error BG0000: System.InvalidOperationException: The XPath expression evaluated to unexpected type System.Boolean.
```

Tried several Xpath patterns:

https://github.com/xamarin/AndroidX/blob/mu-compose/source/androidx.compose.runtime/runtime/Transforms/Metadata.Namespaces.xml#L9-L13

Reply:
It's because they duplicate the `path=` part, which is so subtle I didn't even notice when I copied the value for that comment.  😁

Reply:
I know - staring at the code does not help... Need some time off...

fixed.